### PR TITLE
bug/855_attributes_order_mysql

### PR DESCRIPTION
**Pull request by frbattid**
* Twin of PR https://github.com/telefonicaid/fiware-cygnus/pull/856
* Assignee @pcoello25 but LGTM from @vgarciag  and/or @mcarracedo is also required

### CODE REVIEW

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java [hunk 1]
```diff
@@ -180,19 +182,19 @@ void persistBatch(Batch batch) throws Exception {
...
     
     /**
-     * Class for aggregating fieldValues.
+     * Class for aggregating.
      */
     private abstract class MySQLAggregator {
         
-        // string containing the data fieldValues
-        protected String aggregation;
+        // object containing the aggregted data
```
Comment (pcoello25) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java]: aggregted data -> aggregated data

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java [hunk 2]
```diff
@@ -461,24 +480,24 @@ private MySQLAggregator getAggregator(boolean rowAttrPersistence) {
...
         String dbName = aggregator.getDbName(enableLowercase);
         String tableName = aggregator.getTableName(enableLowercase);
         
         LOGGER.info("[" + this.getName() + "] Persisting data at OrionMySQLSink. Database ("
-                + dbName + "), Table (" + tableName + "), Fields (" + fieldNames + "), Values ("
-                + fieldValues + ")");
+                + dbName + "), Table (" + tableName + "), Fields (" + fieldsForInsert + "), Values ("
+                + valuesForInsert + ")");
         
         // creating the database and the table has only sense if working in row mode, in column node
```
Comment (pcoello25) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java]: column node -> column mode

**Comment by vgarciag** [comment]
LGTM

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java (hunk 2 above)
Comment (frbattid) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/f43ad5018cb279a5da36a91bdd130afdd99590cc

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java (hunk 1 above)
Comment (frbattid) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/f43ad5018cb279a5da36a91bdd130afdd99590cc

**Comment by pcoello25** [comment]
LGTM

**Pull request merged**